tests/unit/gapic/pubsub_v1/test_publisher.py

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
```

Comment:
Unfortunately all these tests under tests/unit/gapic are auto-generated so these changes will be overwritten when the library is regenerated.

Reply:
These changes were added to the auto-generated files in: https://github.com/googleapis/python-pubsub/pull/746